support-models/src/main/scala/com/gu/support/zuora/api/SubscriptionData.scala

```diff
@@ -118,6 +122,7 @@ object ReaderType {
     s match {
       case Gift.value => Gift
       case Agent.value => Agent
+      case Corporate.value => Corporate
       case _ => Direct
```

Comment:
interesting that we assume Direct rather than failing, this would reduce the number of "can't subscribe" incidents but could cause some incorrect data in the long term

Reply:
We actually never deserialise this in the app, we only serialise to send to Zuora but I've added a new `Unknown` value to be doubly safe.

Reply:
consider just deleting the deserialiser if it's unused at present